steering_vectors/steering_vector.py

```diff
@@ -45,13 +45,16 @@ def patch_activations(
         layer_config: Optional[ModelLayerConfig] = None,
         operator: Optional[PatchOperator] = None,
         multiplier: float = 1.0,
-        min_token_index: int = 0,
+        min_token_index: int | None = None,
+        token_indices: list[int] | slice | Tensor | None = None,
```

Comment:
I was going to write here that it's better to use `Optional[types] = None` rather than `types | None = None` since it indicates the intention that it's an optional parameter, but after some searching it looks like I'm wrong on this and the best practice now is to replace all the `Optional[types]` with `types | None`. I added an issue to remove all the `Optional` stuff throughout the codebase here: https://github.com/steering-vectors/steering-vectors/issues/23